test/integration/machine-learning.spec.ts

```diff
@@ -101,6 +105,58 @@ describe('admin.machineLearning', () => {
         });
     });
 
+    it('creates a new Model with valid AutoML TFLite ModelFormat', function () {
+      const modelOptions: admin.machineLearning.ModelOptions = {
+        displayName: 'node-integ-test-create-automl',
+        tags: ['tagAutoml'],
+        tfliteModel: { automlModel: 'this will be replaced below' }
+      };
+      this.timeout(60000); // Allow up to 60 seconds for this test.
```

Comment:
Let's elaborate a bit as to why this takes so long.

Reply:
done